src/components/cc-global-header.vue

```diff
@@ -34,11 +34,14 @@
         :class="{ ['navbar-menu']: true, ['is-active']: isBurgerMenuActive }"
       >
         <div class="navbar-end">
-          <p v-if="errorMessage" class="navbar-item navbar-link is-arrowless">
-            We are having trouble loading the menu
-            <br />
-            Error Message: {{ errorMessage }}.
+          <p v-if="menuLoading" class="navbar-item navbar-link is-arrowless">
+            Loading menu..
```

Comment:
Minor detail, but an ellipsis has three dots. In any case, it may be unnecessary.

```suggestion
            Loading menu.
```